lib/browser/lighthouse.js

```diff
@@ -32,114 +32,138 @@
 
 'use strict';
 
-const assert = require('assertive');
+const assert = require('assert');
 const isEmpty = require('lodash/isEmpty');
+const flatMap = require('lodash/flatMap');
 
+/**
+ * @typedef import('lighthouse/types/lhr') LH
+ */
+
+const defaultConfig = {
+  extends: 'lighthouse:default',
+  settings: {
+    onlyCategories: [],
+    throttlingMethod: 'simulate',
+  },
+};
+
+const defaultFlags = {
+  chromeFlags: [
```

Comment:
could these get re-used from testium-core somehow?

Reply:
it could be possible but there are a few things to consider:
-  we would need to add testium-core to the dependencies. this would create a circular dependency with testium-core
- the flags differ https://github.com/testiumjs/testium-core/blob/master/lib/processes.js#L125-L151